packages/storybook/src/filetypes.d.ts

```diff
@@ -2,3 +2,8 @@ declare module '*.md?raw' {
   const content: string;
   export default content;
 }
+
+declare module '*.json' {
```

Comment:
Is het niet slimmer om `"resolveJsonModule": true` in de `tsconfig.json` (since TS 2.9) te zetten of bijv. `import colors from "./colors.json" with { type: "json" };` in je imports (TS 5.3) te gebruiken?

Reply:
Dat zijn van die dingen die je leert kennen met het opzetten van nieuw project. Iets wat ik heeeeeeeel lang niet meer gedaan heb. Dankjewel voor deze les. Ik heb het aangepast. Daarbij heb ik het (voorlopig) alleen voor `storybook` package aangezet. Mochten we het meer nodig hebben kunnen we het wel verplaatsen naar de globale `tsconfig.json`